docs/modules/layers/normalization.rst

```diff
@@ -13,3 +13,8 @@ Normalization layers
 
 .. autofunction:: batch_norm
 
+.. autoclass:: InstanceNormLayer
+    :members:
+
+.. autofunction:: instance_norm
+
```

Comment:
This doesn't match the code any more. Should have StandardizationLayer, instance_norm and layer_norm instead.